spec/parcels/parcel-error-handlers.spec.js

```diff
@@ -148,11 +148,10 @@ describe('parcel errors', () => {
                 expect(err.message.indexOf(`UNMOUNTING`)).toBeGreaterThan(-1);
                 expect(err.message.indexOf(`unmount-error`)).toBeGreaterThan(-1);
               })
-              return Promise.resolve()
             }).then(() => {
               expect(errs.length).toBe(1);
               expect(errs[0].appName).toBe('app-parcel-unmount-errors');
-              expect(errs[0].message.indexOf(`Application 'app-parcel-unmount-errors' died in status UNMOUNTING: Parcel 'unmount-error' died in status SKIP_BECAUSE_BROKEN: Parcel 'unmount-error' died in status UNMOUNTING: unmount error`)).toBeGreaterThan(-1);
+              expect(errs[0].message.indexOf(`Application 'app-parcel-unmount-errors' died in status NOT_MOUNTED: Parcel 'unmount-error' died in status UNMOUNTING: unmount error`)).toBeGreaterThan(-1);
```

Comment:
I changed this error message in my last PR and now am changing it back -- this new one is correct because there are only two things involved: an app and a parcel. Not an app and two parcels like the changed error message says